spec/spec_helper.rb

```diff
@@ -27,6 +27,6 @@
 end
 Capybara.javascript_driver = :poltergeist
 
-def in_travis?
-  !ENV['TRAVIS'].nil? && ENV['TRAVIS'] == 'true'
+def in_ci?
+  !ENV['CI'].nil? && ENV['CI'] == 'true'
```

Comment:
Maybe these could support the existing environment variables, something like:

```
ENV['CI'] == 'true' || ENV['TRAVIS'] == 'true'
```

Reply:
'CI' supports a CI environment https://circleci.com/docs/2.0/env-vars/#circleci-environment-variable-descriptions. 'CIRCLECI' would support only circleci.